task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -32,10 +32,13 @@ spec:
 
         relImgs="$(yq -r '.relatedImages[]?.image' $catalog)"
         if [ $? -ne 0 ]; then
-          echo "Could not get related images, make sure catalog.yaml exists in FBC fragment image and has valid yaml format."
-          HACBS_TEST_OUTPUT="$(make_result_json -r FAILURE -f 1)"
-          echo "${HACBS_TEST_OUTPUT}" | tee "$(results.HACBS_TEST_OUTPUT.path)"
+          relImgs="$(jq -r '.relatedImages[]?.image' $catalog)"
```

Comment:
can we provide an explanatory comment here, why we are doing such magic with jq?

Reply:
Sounds about right.